lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise_question.rb

```diff
@@ -59,7 +63,7 @@ def bake(question:, number:, only_number_solution:)
             #{context if context.present?}
             #{"<span class='os-divider'>. </span>" if context.present?}
             #{question.stimulus&.cut&.paste}
-            #{question.stem.cut.paste}
+            #{question.stem&.cut&.paste unless alphabetical_multipart}#{question.search(".alphabetical-multipart")&.cut&.paste if alphabetical_multipart}
```

Comment:
On the same line to avoid extra newline?

Reply:
Oh! And to prevent this line from stealing the first question stem and putting it on top of everything else in a multipart question

Reply:
what about 
`#{alphabetical_multipart  ? question.search(".alphabetical-multipart").cut.paste : question.stem.cut.paste}`